pyspeckit/spectrum/models/ammonia.py

```diff
@@ -162,7 +164,7 @@ def ammonia(xarr, tkin=20, tex=None, ntot=14, width=1, xoff_v=0.0,
         these equations, not just the final one)
         """
         dT0 = 41.5                    # Energy diff between (2,2) and (1,1) in K
-        trot = tkin/(1+tkin/dT0*np.log(1+0.6*np.exp(-15.7/tkin)))
+        trot = trot/(1+trot/dT0*np.log(1+0.6*np.exp(-15.7/trot)))
```

Comment:
@keflavich 
I think the original version is correct (i.e., tkin on the right).  Swift et al., 2005 make the relationship in the coupling for a the three state system: (1,1), (2,1), (2,2).
Time to RADEX check this?